test/b2gedmntuples_cfg.py

```diff
@@ -275,18 +258,29 @@
 ### ------------------------------------------------------------------
 from JMEAnalysis.JetToolbox.jetToolbox_cff import jetToolbox
 
+listBtagDiscriminators = [ 'pfTrackCountingHighEffBJetTags',
+		'pfTrackCountingHighPurBJetTags',
+		'pfJetProbabilityBJetTags',
+		'pfJetBProbabilityBJetTags',
+		'pfSimpleSecondaryVertexHighEffBJetTags',
+		'pfSimpleSecondaryVertexHighPurBJetTags',
+		'pfCombinedSecondaryVertexV2BJetTags',
+		'pfCombinedInclusiveSecondaryVertexV2BJetTags',
+		'pfBoostedDoubleSecondaryVertexAK8BJetTags'
+		]
+
```

Comment:
well actually, this is up to the btag experts (I think you :) 
I can drop the ones that you think are not necessary. But is it a big difference in time? Honestly I didn't pay much attention to it because it ran fast in my opinion but I can remove the discriminators that are not store in the Ntuples.